apps/backend/src/users/user.entity.ts

```diff
@@ -1,9 +1,14 @@
-import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+import { ObjectId } from 'mongodb';
+import { Entity, Column, ObjectIdColumn } from 'typeorm';
+import { Status, Role, Team } from './types';
 
 @Entity()
 export class User {
-  @PrimaryGeneratedColumn()
-  id: number;
+  @ObjectIdColumn()
```

Comment:
:wrench: just to make our lives easier, I think we should have this column be a `number` column instead so we're not dealing with too many `ObjectId`s (which can get a little tedious):
```ts
  @Column({ primary: true })
  userId: number;
```